int8 option
--- 
int8 playerCount